cmd/dra-example-kubeletplugin/cdi.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	cdiapi "github.com/container-orchestrated-devices/container-device-interface/pkg/cdi"
 	cdispec "github.com/container-orchestrated-devices/container-device-interface/specs-go"
+
```

Comment:
This is unrelated to the PR, but I agree that this is better, so let's leave this in.

Reply:
Thanks!